apps/admin-server/src/components/ui/sidenav.tsx

```diff
@@ -16,13 +17,23 @@ export function Sidenav({
   className?: string;
   narrow?: boolean;
 }) {
+  const { data } = useSession()
   const router = useRouter();
   const [location, setLocation] = useState('');
 
   useEffect(() => {
     setLocation(router.pathname);
   }, [router]);
 
+  useEffect(() => {
```

Comment:
Zoals in PR https://github.com/openstad/openstad-headless/pull/112 als je een wrapper hook maakt voor session en die useEffect met session [data] dependency, dan op elke plek waar je session en die useEffect gebruikt, dan kan hoef je maar 1 regel code toe te voegen